src/main/java/org/zalando/nakadi/controller/SubscriptionController.java

```diff
@@ -110,17 +115,32 @@ public SubscriptionController(final SubscriptionDbRepository subscriptionReposit
     @RequestMapping(method = RequestMethod.GET)
     public ResponseEntity<?> listSubscriptions(
             @Nullable @RequestParam(value = "owning_application", required = false) final String owningApplication,
+            @Nullable @RequestParam(value = "event_type", required = false) final Set<String> eventTypes,
+            @RequestParam(value = "limit", required = false, defaultValue = "20") final int limit,
+            @RequestParam(value = "offset", required = false, defaultValue = "0") final int offset,
             final NativeWebRequest request) {
 
         if (!featureToggleService.isFeatureEnabled(HIGH_LEVEL_API)) {
             return new ResponseEntity<>(NOT_IMPLEMENTED);
         }
 
+        if (limit < 1 || limit > 1000) {
```

Comment:
I checked that. Theoretically it is possible using [MethodValidationPostProcessor](http://docs.spring.io/autorepo/docs/spring-framework/4.1.8.RELEASE/javadoc-api/org/springframework/validation/beanvalidation/MethodValidationPostProcessor.html). But it will not work in our controller test setup :(
So I will keep it as it is.